ledger/src/lib.rs

```diff
@@ -161,13 +163,16 @@ impl<N: Network, C: ConsensusStorage<N>> Ledger<N, C> {
         let vm = VM::from(store)?;
         lap!(timer, "Initialize a new VM");
 
+        let current_committee = vm.finalize_store().committee_store().current_committee().ok();
```

Comment:
Not saying we do this, but technically we can get rid of the `Option` around the `Committee` cache since we ensure that the genesis block must be added to the ledger and already contains a `Committee`. This means we can directly extract the committee from the `Ratify::Genesis` for the case where Ledger starts as empty.

Something like 
```
let current_committee = ledger.vm.block_store().heights().max() {
  None => // Extract committee from genesis block.
  Some(_) => vm.finalize_store().committee_store().current_committee()?
};
```
Though, the current approach is likely cleaner.